tasks/update_history.ps1

```diff
@@ -58,7 +67,11 @@ $Searcher = $Session.CreateUpdateSearcher()
 # Returns IUpdateSearcher https://msdn.microsoft.com/en-us/library/windows/desktop/aa386515(v=vs.85).aspx
 
 $historyCount = $Searcher.GetTotalHistoryCount()
-$Searcher.QueryHistory(0, $historyCount) | ForEach-Object -Process {
+if ($historyCount -gt $MaximumUpdates) { $historyCount = $MaximumUpdates }
+$Searcher.QueryHistory(0, $historyCount) |
+  Where-Object { ($Title -eq '') -or ($_.Title -match $Title) } |
```

Comment:
For these two lines do we want to use `[string]::IsNullOrEmpty($Variable)` instead of `$Variable -eq ''`? Then I think we don't have to set an initial value in the param block.

Reply:
Fixed